package.json

```diff
@@ -3,13 +3,21 @@
   "svelte": "dist/index.js",
   "exports": {
     ".": {
-      "svelte": "./dist/index.js"
+      "svelte": "./dist/index.js",
+      "types": "./dist/ts/index.d.ts"
     }
   },
   "module": "dist/index.mjs",
   "main": "dist/index.umd.js",
   "types": "dist/ts/index.d.ts",
-  "author": "Paolo Tiu <paolotiu17@gmail.com>",
+  "contributors": [
+    {"name": "Paolo Tiu",
+      "email": "paolotiu17@gmail.com"
+    },
+    {"name": "Justus Perlwitz",
+      "email": "hello@justus.pw"
+    }
+  ],
```

Comment:
Perhaps I'm not aware of some offline discussion on repo ownership, but curious if this is acceptable to the original maintainer. If not, are you hosting a fork anywhere?

Reply:
Thanks for this @dusty-phillips!! I'll resolve some of these issues then merge the pr